catalog/ui/search-ui/standard/src/main/webapp/js/view/MetacardDetail.view.js

```diff
@@ -92,6 +104,11 @@ define([
                 }
             },
             updateScrollbar: function () {
+
+               $('.tab-content',this.$el).perfectScrollbar({
```

Comment:
Don't use jquery directly here. Do this.$ and you won't need the extra argument.